frontend/public/metalkube/components/host/host.tsx

```diff
@@ -55,7 +62,21 @@ const HostRow = ({ obj: host }) => {
     },
   } = host;
 
-  const machineName = _.get(host, 'status.machineRef.name');
+  const machineName = _.get(host, 'spec.machineRef.name');
```

Comment:
I'll replace this with a selector call once https://github.com/kubevirt/web-ui-components/pull/333 merges

Reply:
Done